CONTRIBUTING.md

```diff
@@ -163,16 +163,20 @@ like [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv),
 [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/), and
 [pyenv-virtualenvwrapper](https://github.com/pyenv/pyenv-virtualenvwrapper).
 
+for git related issues refer to https://github.com/101Loop/APIManager-Flutter/blob/master/CONTRIBUTING.md
```

Comment:
@HardipinderS I told you to take reference from that link and update this contributing guide on how to work on a forked version of this copy. So, please update the documentation guide how a user can contribute using a forked copy of this repo. We don't want users redirect to some other pages.

Reply:
I'm not sure what and how much exactly is to be referred and added in that part, so gave the link. i would request you to please look into that part, as I'm not that sure what exactly is required